fuse/daemon.cpp

```diff
@@ -128,6 +128,13 @@ void Job::dump() {
 
 	first = true;
 	for (auto &x : files_wrote) {
+		// files prefixed with .fuse_hidden are implementation details of libfuse
+		// and should not be treated as outputs.
+		// see: https://github.com/libfuse/libfuse/blob/fuse-3.10.3/include/fuse.h#L161-L177
+		const std::string prefix = ".fuse_hidden";
```

Comment:
Please move this string outside of the loop (otherwise you are allocating + freeing it on each iteration).